src/cloudsc_loki/CMakeLists.txt

```diff
@@ -619,6 +619,10 @@ if( HAVE_CUDA )
             ${COMMON_MODULE}/yomphyder.F90
             ${COMMON_MODULE}/yoethf.F90
             ${COMMON_MODULE}/yoecldp.F90
+            ${COMMON_MODULE}/cloudsc_field_state_mod.F90
+            ${COMMON_MODULE}/cloudsc_flux_type_mod.F90
+            ${COMMON_MODULE}/cloudsc_aux_type_mod.F90
+            ${COMMON_MODULE}/cloudsc_state_type_mod.F90
```

Comment:
This should probably not be added here, since this is the SCC CUF variant that is not based on the field variant of the gpu driver. My thinking is that a new loki_transform will be added through the dwarf-p-cloudsc/je-field-api-offload branch that I am using for development of the Loki Field API offload transformation.